### PR TITLE
Add disney domains

### PR DESCRIPTION
**Pull request by yaotthaha**
之前号被GitHub封了，重新开pull request

欢迎测试

### CODE REVIEW

File: data/disney [hunk 1]
```diff
@@ -121,3 +138,4 @@ streamingdisney.net
 thestationbymaker.com
 thisispolaris.com
 watchdisneyfe.com
+
```
Comment (IceCodeNew) [inline data/disney:141]: You have introduced a redundant empty line here.

File: data/disney [hunk 2]
```diff
@@ -88,22 +97,30 @@ disneyme.com
...
 disneystreaming.com
+disneystreaming.com
 disneysubscription.com
 disneytickets.co.uk
 disneyturkiye.com.tr
 disneytvajobs.com
 disneyworld-go.com
 dssott.com
-go-disneyworldgo.com
+dssott.com
```
Comment (IceCodeNew) [inline data/disney:120]: This line seems redundant.

**Review by IceCodeNew — CHANGES_REQUESTED** [review]

**Comment by IceCodeNew** [comment]
Request other reviewers based on GitHub's recommendations. I don't have much experience dealing with the `Disney` sites.

File: data/disney [hunk 3]
```diff
@@ -88,22 +97,30 @@ disneyme.com
...
 go.com
+go-disneyworldgo.com
+js-agent.newrelic.com
```
Comment (AkinoKaede) [inline data/disney:123]: Although Disney is a customer of New Relic, I do not think it a domain of Disney.

File: data/disney [hunk 4]
```diff
@@ -47,10 +47,18 @@ disney.sg
...
 20thcenturystudios.jp
+adobedtm.com
 adventuresbydisney.com
 babble.com
 babyzone.com
+bamgrid.com
+bam.nr-data.net
 beautyandthebeastmusical.co.uk
+braze.com
+cdn.optimizely.com
```
Comment (AkinoKaede) [inline data/disney:58]: I think Optimizely CDN also provide services for others.

File: data/disney [hunk 5]
```diff
@@ -47,10 +47,18 @@ disney.sg
...
+cdn.registerdisney.go.com
+cws.conviva.com
```
Comment (AkinoKaede) [inline data/disney:60]: This is a domain owned by Conviva that offering solutions for online video optimization and online video analytics. Can it prove that this domain only provides services to Disney?

File: data/disney [hunk 6]
```diff
@@ -47,10 +47,18 @@ disney.sg
...
 20thcenturystudios.com.br
 20thcenturystudios.jp
+adobedtm.com
 adventuresbydisney.com
 babble.com
 babyzone.com
+bamgrid.com
+bam.nr-data.net
 beautyandthebeastmusical.co.uk
+braze.com
```
Comment (AkinoKaede) [inline data/disney:57]: It is owned by Braze, they also provides services for other companies, such as HBO and Etsy.

**Review by AkinoKaede — CHANGES_REQUESTED** [review]

File: data/disney [hunk 7]
```diff
@@ -47,10 +47,18 @@ disney.sg
...
+cdn.optimizely.com
+cdn.registerdisney.go.com
+cws.conviva.com
+d9.flashtalking.com
```
Comment (AkinoKaede) [inline data/disney:61]: I don’t know if this is a domain that only serves Disney

File: data/disney [hunk 8]
```diff
@@ -54,10 +54,6 @@ babyzone.com
 bamgrid.com
 bam.nr-data.net
 beautyandthebeastmusical.co.uk
-braze.com
-cdn.optimizely.com
-cdn.registerdisney.go.com
```
Comment (AkinoKaede) [inline data/disney]: Sorry, I chose the wrong line. `cdn.registerdisney.go.com` should not be removed.

File: data/disney (hunk 7 above)
Comment (AkinoKaede) [reply data/disney:58]: @IceCodeNew How do you think about `d9.flashtalking.com`?

File: data/disney [hunk 9]
```diff
@@ -54,6 +54,7 @@ babyzone.com
 bamgrid.com
 bam.nr-data.net
 beautyandthebeastmusical.co.uk
+cdn.registerdisney.go.com
```
Comment (AkinoKaede) [inline data/disney:57]: It should be removed, `go.com` included this.

File: data/disney [hunk 10]
```diff
@@ -47,10 +47,15 @@ disney.sg
 20thcenturystudios.com.au
 20thcenturystudios.com.br
 20thcenturystudios.jp
+adobedtm.com
 adventuresbydisney.com
 babble.com
 babyzone.com
+bamgrid.com
+bam.nr-data.net
```
Comment (AkinoKaede) [inline data/disney:55]: It a tracker by New Relic

**Review by AkinoKaede — CHANGES_REQUESTED** [review]

File: data/disney (hunk 7 above)
Comment (AkinoKaede) [reply data/disney:58]: I found a JavaScript `http://d9.flashtalking.com/d9core`, and I think it serves many advertisers.

File: data/disney [hunk 11]
```diff
@@ -47,10 +47,15 @@ disney.sg
 20thcenturystudios.com.au
 20thcenturystudios.com.br
 20thcenturystudios.jp
+adobedtm.com
```
Comment (Loyalsoldier) [inline data/disney:50]: It belongs to Adobe.

File: data/disney [hunk 12]
```diff
@@ -65,6 +70,7 @@ disneycareers.com
 disneychannelonstage.com
 disneychannelroadtrip.com
 disneycruisebrasil.com
+disney.demdex.net
```
Comment (Loyalsoldier) [inline data/disney:73]: Also Adobe's.

File: data/disney [hunk 13]
```diff
@@ -88,22 +94,28 @@ disneyme.com
...
+disneyplus.bn5x.net
 disneyplus.com
+disneyplus.com
+disneyplus.com.ssl.sc.omtrdc.net
+disney-portal.my.onetrust.com
 disneyredirects.com
 disneysrivieraresort.com
 disneystore.com
 disneystreaming.com
+disneystreaming.com
```
Comment (Loyalsoldier) [inline data/disney:110]: Duplicated.

**Comment by Loyalsoldier** [comment]
Please remove domains that do NOT belong to Disney.

File: data/disney (hunk 7 above)
Comment (IceCodeNew) [reply data/disney:54]: > I found a JavaScript http://d9.flashtalking.com/d9core, and I think it serves many advertisers.

thanks for verifying, @yaotthaha would you please remove this line?

File: data/disney [hunk 14]
```diff
@@ -103,7 +108,6 @@ disneytvajobs.com
 disneyworld-go.com
 dssott.com
 go-disneyworldgo.com
-go.com
```
Comment (IceCodeNew) [inline data/disney:106]: What happens to `cdn.registerdisney.go.com`?
And why is the whole `go.com` being deleted?

**Review by IceCodeNew — CHANGES_REQUESTED** [review]

File: data/disney [hunk 15]
```diff
@@ -47,14 +47,10 @@ disney.sg
 20thcenturystudios.com.au
 20thcenturystudios.com.br
 20thcenturystudios.jp
-adobedtm.com
 adventuresbydisney.com
 babble.com
 babyzone.com
-bamgrid.com
```
Comment (AkinoKaede) [inline data/disney]: Sorry, I means `bam.nr-data.net` is not owned by Disney, but `bamgrid.com` is. It should be readded.

File: data/disney (hunk 14 above)
Comment (IceCodeNew) [reply data/disney:106]: > What happens to `cdn.registerdisney.go.com`?
> And why is the whole `go.com` being deleted?

I need to know the answer before I give this pr approval.

**Comment by yaotthaha** [comment]
Sorry, I'm in China. You know github is broken in China incompletely and I made some mistake...

**Comment by IceCodeNew** [comment]
> Sorry, I'm in China. You know github is broken in China incompletely and I made some mistake. But now, everything is ok.

image

There is still a remaining question.

**Comment by yaotthaha** [comment]
Everything has been done. Waiting someone could test it.

**Review by AkinoKaede — APPROVED** [review]

File: data/disney [hunk 16]
```diff
@@ -88,10 +89,14 @@ disneyme.com
...
 disneymovieinsiders.com
 disneymusicpromotion.com
+disney.my.sentry.io
 disneynewseries.com
 disneynow.com
 disneypeoplesurveys.com
+disneyplus.bn5x.net
 disneyplus.com
+disneyplus.com.ssl.sc.omtrdc.net
+disney-portal.my.onetrust.com
```
Comment (KevinZonda) [inline data/disney:99]: Hummm, I'm not sure about the order of `.`, `-` and `p`.
/cc @IceCodeNew

**Review by KevinZonda — CHANGES_REQUESTED** [review]

**Review by IceCodeNew — APPROVED** [review]

**Review by KevinZonda — APPROVED** [review]

**Comment by KevinZonda** [comment]
Thank you guys for your hard working!

**Pull request merged**